pkg/build/scripts/users/add-users.sh.tpl

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+# Without this, the script will run successfully during combustion, but when /home
+# is mounted it will hide the /home used during these user creations.
+mount /home
+
+{{ range . }}
+
+{{/* Non-root users */}}
+{{ if (ne .Username "root") }}
+  useradd -m {{.Username}}
+  {{ if .Password }}
+    echo '{{.Username}}:{{.Password}}' | chpasswd -e
+  {{ end }}
+  {{ if .SSHKey }}
+    mkdir -pm700 /home/{{.Username}}/.ssh/
+    echo '{{.SSHKey}}' >> /home/{{.Username}}/.ssh/authorized_keys
+    chown -R {{.Username}} /home/{{.Username}}/.ssh
+  {{ end }}
+{{ end }}
+
+{{/* Root */}}
+{{ if (eq .Username "root") }}
```

Comment:
Is there a particular reason not to `{{ else }}` instead?

Reply:
Nothing specific. My head was in a different place editing a template v. code so I just wasn't thinking of it as a if/else construct so much as a flag for inclusion (not sure if that made sense).